### PR TITLE
[FR] Update `Client\Beatmap_editor\Song_Setup`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page `Client\Beatmap_editor\Song_Setup`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 1]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
+Dans osu!, le système de timing ne permet pas à un objet d'être frapper tant que l'objet précédent n'a pas été frapper et n'a pas reçu de jugement, ou tant qu'il n'a pas manqué après que la fenêtre de hit soit passée. Si les fenêtres de hits sont trop longues, la fenêtre de hit de l'objet peut chevaucher la fenêtre précédente. Pendant ce temps, l'objet est "verrouillé" et les coups ne comptent pas. Ce phénomène est appelé de manière informelle "notelock" dans osu!, ou "column locking" ou "cbrushing" dans osu!mania. Pour plus de détails et d'exemples, voir l'[article sur le notelock](/wiki/Gameplay/Judgement/Notelock). Pour cette raison, il est conseillé de ne pas utiliser de faibles valeurs d'OD dans les maps à fort BPM.
```
Comment (Biokax) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:143]:
```suggestion
Dans osu!, le système de timing ne permet pas à un objet d'être touché tant que l'objet précédent ne l'a pas été et n'a pas reçu de jugement, ou tant qu'il n'a pas manqué après que la fenêtre de hit soit passée. Si les fenêtres de hits sont trop longues, la fenêtre de hit de l'objet peut chevaucher la fenêtre précédente. Pendant ce temps, l'objet est "verrouillé" et les coups ne comptent pas. Ce phénomène est appelé de manière informelle "notelock" dans osu!, ou "column locking" ou "cbrushing" dans osu!mania. Pour plus de détails et d'exemples, voir l'[article sur le notelock](/wiki/Gameplay/Judgement/Notelock). Pour cette raison, il est conseillé de ne pas utiliser de faibles valeurs d'OD dans les maps à fort BPM.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by Biokax — APPROVED** [review]

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 2]
```diff
@@ -250,7 +252,7 @@ En déplaçant le slider vers la droite, vous espacerez la "proximité" des obje
 
 *Remarque : Les cercles empilés dans osu! se déplaceront vers le point où l'empilement se produit.*
 
-Les objets sont considérés comme appartenant à la même pile s'ils se trouvent à la même coordonnée et pas plus loin dans le temps que `[(Fenêtre d'approche) * (Limite de pile) / 10]`, où `Fenêtre d'approche` est la durée pendant laquelle les objets restent à l'écran (voir [Approach Rate](#approach-rate).
+Les objets sont considérés comme appartenant à la même pile s'ils se trouvent à la même coordonnée et pas plus loin dans le temps que `[(Fenêtre d'approche) * (Limite de pile) / 10]`, où `Fenêtre d'approche` est la durée pendant laquelle les objets restent à l'écran (voir [La section sur l'approach rate](#approach-rate).
```
Comment (Walavouchey) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:255]: here it's "approach rate", while elsewhere you've changed it to "Approach rate". In english, these have slowly been transitioning towards using all lowercase when not written as an acronym, but should they have the first letter in capital in french?

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 3]
```diff
@@ -81,7 +76,7 @@ Les paramètres de difficulté de la beatmap ; chaque paramètre affecte différ
 
 *Note : En maintenant la touche `Shift` enfoncée, on peut obtenir un ajustement précis de 0,1 par rapport à l'ajustement standard de 1.*
 
-La notation courante pour représenter un paramètre de difficulté et sa valeur est l'abréviation du paramètre suivie de la valeur du paramètre. Par exemple, `CS4` et `HP4.3` seront une beatmap avec un Circle Size de 4 et un taux de drain de santé de 4.3 respectivement. En théorie, les paramètres de difficulté négatifs peuvent ressembler à `AR-2` pour un Approach Rate de -2.
+La notation courante pour représenter un paramètre de difficulté et sa valeur est l'abréviation du paramètre suivie de la valeur du paramètre. Par exemple, `CS4` et `HP4.3` seront une beatmap avec un Circle size de 4 et un taux de drain de santé de 4.3 respectivement. En théorie, les paramètres de difficulté négatifs peuvent ressembler à `AR-2` pour un Approach rate de -2.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:79]:
```suggestion
La notation courante pour représenter un paramètre de difficulté et sa valeur est l'abréviation du paramètre suivie de la valeur du paramètre. Par exemple, `CS4` et `HP4.3` seront une beatmap avec un circle size de 4 et un taux de drain de santé de 4.3 respectivement. En théorie, les paramètres de difficulté négatifs peuvent ressembler à `AR-2` pour un approach rate de -2.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 4]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
 
 *Note : Plus d'HP sont récupérés avec les scores [Geki](/wiki/Gameplay/Judgement/Geki) et [Katu](/wiki/Gameplay/Judgement/Katu), qui peuvent être donnés par le dernier objet dans un combo.*
 
-### Circle Size
+### Circle size
 
-![Aperçu des différentes Circle Size](img/Circle_sizes.png "Aperçu des différentes Circle Size")
+![Aperçu des différentes Circle size](img/Circle_sizes.png "Aperçu des différentes Circle size")
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:91]:
```suggestion
![Aperçu des différentes circle size](img/Circle_sizes.png "Aperçu des différentes Circle size")
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 5]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
...
 
 *Note : Cette option est visible dans osu!taiko, mais elle n'affecte pas le gameplay.*
 
-Le **Circle Size**, communément abrégée en CS, n'existe que dans osu! et osu!catch. CS détermine la taille des objets ou, plus précisément dans osu!catch, détermine la taille de l'attrapeur et du fruit. Les valeurs vont de 2 à 7 ; plus les valeurs CS sont élevées, plus les objets sont petits. En utilisant la valeur la plus élevée, les objets seront aussi grands que le curseur.
+Le **Circle size**, communément abrégée en CS, n'existe que dans osu! et osu!catch. CS détermine la taille des objets ou, plus précisément dans osu!catch, détermine la taille de l'attrapeur et du fruit. Les valeurs vont de 2 à 7 ; plus les valeurs CS sont élevées, plus les objets sont petits. En utilisant la valeur la plus élevée, les objets seront aussi grands que le curseur.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:95]:
```suggestion
Le **circle size**, communément abrégée en CS, n'existe que dans osu! et osu!catch. CS détermine la taille des objets ou, plus précisément dans osu!catch, détermine la taille de l'attrapeur et du fruit. Les valeurs vont de 2 à 7 ; plus les valeurs CS sont élevées, plus les objets sont petits. En utilisant la valeur la plus élevée, les objets seront aussi grands que le curseur.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 6]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
...
 
 La plupart des beatmaps utilisent une valeur de 3 ou 4, tandis qu'une valeur de 5 ou 6 est parfois utilisée et une valeur de 7 n'est presque jamais utilisée. Une valeur de 7 et plus est susceptible d'être vue lors de l'utilisation du mod [Hard Rock](/wiki/Game_modifier/Hard_Rock).
 
 Le réglage de la valeur CS à une valeur inférieure à 2 ou supérieure à 7 peut être fait en modifiant manuellement le fichier `.osu`.
 
 #### Key Count
 
-Key Count, également connu sous le nom de Keys, remplace la métrique [Circle Size](#circle-size) pour les beatmaps [osu!mania-specific](#mode-specific). Key Count est auto-explicatif ; il définit le nombre de touches utilisé. Les valeurs vont de 1 à 9 touches.
+Key Count, également connu sous le nom de Keys, remplace la métrique [Circle size](#circle-size) pour les beatmaps [osu!mania-specific](#mode-specific). Key Count est auto-explicatif ; il définit le nombre de touches utilisé. Les valeurs vont de 1 à 9 touches.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:103]:
```suggestion
Key Count, également connu sous le nom de Keys, remplace la métrique [circle size](#circle-size) pour les beatmaps [osu!mania-specific](#mode-specific). Key Count est auto-explicatif ; il définit le nombre de touches utilisé. Les valeurs vont de 1 à 9 touches.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 7]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
...
+### Approach rate
 
 *Page principale : [Approach rate](/wiki/Beatmapping/Approach_rate)*
 
 ![Tableau détaillé de l'AR](/wiki/shared/ARTable.jpg "Tableau détaillé de l'AR")
 
 *Remarque : cette option est visible dans osu!taiko et osu!mania, mais elle n'a aucune incidence sur la jouabilité. C'est principalement parce que les deux utilisent une vitesse de défilement, qui est basée sur le BPM (battements par minute)..*
 
-L'**Approach Rate**, communément abrégée en AR, n'existe que dans osu! et osu!catch. Les valeurs vont de 0 à 10 ; plus les valeurs d'AR sont élevées, plus la hit windows des objets est courte. L'AR indique combien de temps les objets resteront à l'écran, depuis le moment où ils apparaissent jusqu'au moment où il faut les frapper/récupérer.
+L'**Approach rate**, communément abrégée en AR, n'existe que dans osu! et osu!catch. Les valeurs vont de 0 à 10 ; plus les valeurs d'AR sont élevées, plus la hit windows des objets est courte. L'AR indique combien de temps les objets resteront à l'écran, depuis le moment où ils apparaissent jusqu'au moment où il faut les frapper/récupérer.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:115]:
```suggestion
L'**approach rate**, communément abrégée en AR, n'existe que dans osu! et osu!catch. Les valeurs vont de 0 à 10 ; plus les valeurs d'AR sont élevées, plus la hit windows des objets est courte. L'AR indique combien de temps les objets resteront à l'écran, depuis le moment où ils apparaissent jusqu'au moment où il faut les frapper/récupérer.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 8]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
...
 
-Un approach rate plus élevé signifie que les objets seront affichés pendant une période plus courte, ce qui laisse moins de temps de réaction pour planifier. Inversement, un approach rate plus faible signifie que les objets seront affichés à l'écran pendant plus longtemps, ce qui laisse plus de temps pour réagir, mais peut entraîner l'apparition d'un nombre excessif d'objets à l'écran en même temps si l'approach rate est trop faible.
+Un Approach rate plus élevé signifie que les objets seront affichés pendant une période plus courte, ce qui laisse moins de temps de réaction pour planifier. Inversement, un Approach rate plus faible signifie que les objets seront affichés à l'écran pendant plus longtemps, ce qui laisse plus de temps pour réagir, mais peut entraîner l'apparition d'un nombre excessif d'objets à l'écran en même temps si l'Approach rate est trop faible.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:117]:
```suggestion
Un approach rate plus élevé signifie que les objets seront affichés pendant une période plus courte, ce qui laisse moins de temps de réaction pour planifier. Inversement, un approach rate plus faible signifie que les objets seront affichés à l'écran pendant plus longtemps, ce qui laisse plus de temps pour réagir, mais peut entraîner l'apparition d'un nombre excessif d'objets à l'écran en même temps si l'approach rate est trop faible.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 9]
```diff
@@ -91,41 +86,41 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
...
 
 ---
 
 La durée d'un objet qui reste visible à l'écran (sans mods) varie de 1800ms (AR0) à 450ms (AR10).
 
-Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont activés :
+Il existe quatre mods qui modifient le timing de l'Approach rate lorsqu'ils sont activés :
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:123]:
```suggestion
Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont activés :
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 10]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
 
-Il existe quatre mods qui modifient le timing de l'overall difficulty lorsqu'ils sont activés :
+Il existe quatre mods qui modifient le timing de l'Overall difficulty lorsqu'ils sont activés :
 
 - [Easy](/wiki/Game_modifier/Easy): Divise par deux la valeur de l'OD.
 - [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multiplie la valeur de l'OD par 1,4, jusqu'à OD10 au maximum.
-- [Double Time](/wiki/Game_modifier/Double_Time): La valeur OD n'est pas affectée, mais en raison de l'augmentation de 50 % de la vitesse de lecture, les hit windows sont 33 % plus courtes.
-- [Half Time](/wiki/Game_modifier/Half_Time): La valeur OD n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse de lecture, les hit windows sont 33 % plus longues.
+- [Double Time](/wiki/Game_modifier/Double_Time): La valeur de l'OD n'est pas affectée, mais en raison de l'augmentation de 50 % de la vitesse, les fenêtres de hits sont effectivement 33 % plus courtes.
+- [Half Time](/wiki/Game_modifier/Half_Time): La valeur de l'OD n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse, les fenêtres de tir sont effectivement 33 % plus longues.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:162]:
```suggestion
- [Half Time](/wiki/Game_modifier/Half_Time): La valeur de l'OD n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse, les fenêtres de hits sont effectivement 33 % plus longues.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md (hunk 2 above)
Comment (DoryanR) [reply wiki/Client/Beatmap_editor/Song_Setup/fr.md:255]: It's not intentional, no, I thought I had done the opposite (put them in lower case), I rectify that

Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:255]:
```suggestion
Les objets sont considérés comme appartenant à la même pile s'ils se trouvent à la même coordonnée et pas plus loin dans le temps que `[(Fenêtre d'approche) * (Limite de pile) / 10]`, où `Fenêtre d'approche` est la durée pendant laquelle les objets restent à l'écran (voir [la section sur l'approach rate](#approach-rate).
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 11]
```diff
@@ -91,33 +86,33 @@ Dans osu! et osu!catch, cela affecte également les [HP passifs](/wiki/Client/In
 
 *Note : Plus d'HP sont récupérés avec les scores [Geki](/wiki/Gameplay/Judgement/Geki) et [Katu](/wiki/Gameplay/Judgement/Katu), qui peuvent être donnés par le dernier objet dans un combo.*
 
-### Circle Size
+### Circle size
 
-![Aperçu des différentes Circle Size](img/Circle_sizes.png "Aperçu des différentes Circle Size")
+![Aperçu des différentes circle size](img/Circle_sizes.png "Aperçu des différentes Circle size")
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:91]:
```suggestion
![Aperçu des différentes circle size](img/Circle_sizes.png "Aperçu des différentes circle size")
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 12]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
+
+*Page principale : [Overall difficulty](/wiki/Beatmapping/Overall_difficulty)*
 
 ![Tableau détaillé de l'OD](img/ODTable.png "Tableau détaillé de l'OD")
 
-*Remarque : Sur la page de la beatmap, l'overall difficulty est indiquée sous le nom de Précision.*
+*Remarque : le tableau ci-dessus s'applique uniquement au mode de jeu osu!.*\
+*Remarque : Sur la page de la beatmap, l'Overall difficulty est indiquée sous le nom de Précision.*
+
+L'Overall difficulty, communément abrégée en OD, définit le degré de difficulté à atteindre une grande précision. Les valeurs vont de 0 à 10, les valeurs d'OD les plus élevées nécessitant plus de précision et d'exactitude. Étant donné que la précision est importante pour gagner des HP, l'Overall difficulty influence indirectement la difficulté de passer la beatmap. Des valeurs OD plus élevées signifient des fenêtres de hit plus courtes pour clear les objets, à la fois en général et en termes d'obtention de la valeur de score la plus élevée de l'objet.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:141]:
```suggestion
L'overall difficulty, communément abrégée en OD, définit le degré de difficulté à atteindre une grande précision. Les valeurs vont de 0 à 10, les valeurs d'OD les plus élevées nécessitant plus de précision et d'exactitude. Étant donné que la précision est importante pour gagner des HP, l'Overall difficulty influence indirectement la difficulté de passer la beatmap. Des valeurs OD plus élevées signifient des fenêtres de hit plus courtes pour clear les objets, à la fois en général et en termes d'obtention de la valeur de score la plus élevée de l'objet.
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 13]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
+Effets supplémentaires de l'augmentation de l'OD dans chaque mode de jeu :
 
-*Note : Cela signifie que si l'on augmente trop l'OD, il peut être presque impossible d'éliminer dans les temps des spinners auparavant possibles.*
+- osu! : Les spinners doivent être tournés davantage pour remplir la jauge, au point de devenir presque impossible à terminer dans les temps.
+- osu!taiko : Le denden (spinner) nécessite plus de coups pour être éliminé.
+- osu!mania et osu!catch ne sont pas affectés par l'OD.
 
-Dans osu!, le système de timing ne permet pas de toucher un objet tant que l'objet précédent n'a pas été frappé (ou manqué après le passage de la hit windows), ce qui entraîne un échec. Avec une faible valeur d'OD, la hit window d'un objet peut se chevaucher avec celle du suivant. Ainsi, un joueur peut toucher le deuxième objet avec un timing parfait après avoir échoué à toucher le premier objet, mais il manquera complètement les deux objets parce que la hit window du premier objet sera active.
+Pour calculer les fenêtres de hits, voir les formules de l'[article sur l'Overall difficulty](/wiki/Beatmapping/Overall_difficulty).
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:151]:
```suggestion
Pour calculer les fenêtres de hits, voir les formules de l'[article sur l'overall difficulty](/wiki/Beatmapping/Overall_difficulty).
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 14]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
 
 ---
 
-À OD0, un joueur peut obtenir un score de 300 en étant à moins de 79,5 ms de la frappe exacte à temps. À l'autre extrémité de l'échelle, OD10 exige d'être à moins de 19,5 ms pour obtenir le même score de 300.
+À OD 0, un joueur peut obtenir un score de 300 en étant à moins de 79,5 ms de la frappe exacte à temps. À l'autre extrémité de l'échelle, OD 10 exige d'être à moins de 19,5 ms pour obtenir le même score de 300.
 
-Il existe quatre mods qui modifient le timing de l'overall difficulty lorsqu'ils sont activés :
+Il existe quatre mods qui modifient le timing de l'Overall difficulty lorsqu'ils sont activés :
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:157]:
```suggestion
Il existe quatre mods qui modifient le timing de l'overall difficulty lorsqu'ils sont activés :
```

**Comment by Walavouchey** [comment]
forgot OD

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 15]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
 
 Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur réelle de l'AR, la différence de vitesse entraîne un changement d'AR apparent, aussi bas que AR-5 ou aussi élevé que AR11. Dans le tableau d'AR fourni, ces valeurs apparentes sont données pour permettre une comparaison facile entre les vitesses d'approche avec et sans mods. L'AR réel, cependant, est toujours un nombre compris entre 0 et 10. Notez que les niveaux d'AR s'échelonnent de 120 ms pour une valeur inférieure à l'AR5 et de 150 ms pour une valeur supérieure à l'AR5.
 
-### Overall Difficulty
+### Overall difficulty
+
+*Page principale : [Overall difficulty](/wiki/Beatmapping/Overall_difficulty)*
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:134]:
```suggestion
*Page principale : [overall difficulty](/wiki/Beatmapping/Overall_difficulty)*
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 16]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
 
 ![Tableau détaillé de l'OD](img/ODTable.png "Tableau détaillé de l'OD")
 
-*Remarque : Sur la page de la beatmap, l'overall difficulty est indiquée sous le nom de Précision.*
+*Remarque : le tableau ci-dessus s'applique uniquement au mode de jeu osu!.*\
+*Remarque : Sur la page de la beatmap, l'Overall difficulty est indiquée sous le nom de Précision.*
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:139]:
```suggestion
*Remarque : Sur la page de la beatmap, l'overall difficulty est indiquée sous le nom de Précision.*
```

File: wiki/Client/Beatmap_editor/Song_Setup/fr.md [hunk 17]
```diff
@@ -134,30 +129,37 @@ Il existe quatre mods qui modifient le timing de l'approach rate lorsqu'ils sont
...
+
+L'overall difficulty, communément abrégée en OD, définit le degré de difficulté à atteindre une grande précision. Les valeurs vont de 0 à 10, les valeurs d'OD les plus élevées nécessitant plus de précision et d'exactitude. Étant donné que la précision est importante pour gagner des HP, l'Overall difficulty influence indirectement la difficulté de passer la beatmap. Des valeurs OD plus élevées signifient des fenêtres de hit plus courtes pour clear les objets, à la fois en général et en termes d'obtention de la valeur de score la plus élevée de l'objet.
```
Comment (DoryanR) [inline wiki/Client/Beatmap_editor/Song_Setup/fr.md:141]:
```suggestion
L'overall difficulty, communément abrégée en OD, définit le degré de difficulté à atteindre une grande précision. Les valeurs vont de 0 à 10, les valeurs d'OD les plus élevées nécessitant plus de précision et d'exactitude. Étant donné que la précision est importante pour gagner des HP, l'overall difficulty influence indirectement la difficulté de passer la beatmap. Des valeurs OD plus élevées signifient des fenêtres de hit plus courtes pour clear les objets, à la fois en général et en termes d'obtention de la valeur de score la plus élevée de l'objet.
```

**Comment by DoryanR** [comment]
Done

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**